### ISSUE TITLE
3D on Lizmap

### ISSUE THREAD

**Issue by Abelon38** [3liz/lizmap-web-client#374]
Hi there!

Is there documentation on how to add 3D visualization from qgis2threejs onto Lizmap 3.02? 

Thanks for your time!
Abel

**Comment by jlstanus** [comment]
Now with the implementation of 3D on qgis 3. LWC team does have any idea how to use it in LWC?

**Comment by rldhont** [comment]
We need a 3D web client like iTowns http://www.itowns-project.org/ and some 3D vector services (I don't know if QGIS Server will be able to provide 3D GML).

**Comment by jlstanus** [comment]
Yes in fact, this webclient seems to be multi extension. It's a über threeJS with testosterone :-D

i don't know but CityGML looks to be implemented in the desktop version. 
https://twitter.com/lutraconsulting/status/918209161468502017

**Comment by rmarzocchi84** [comment]
> Hi there!
> 
> Is there documentation on how to add 3D visualization from qgis2threejs onto Lizmap 3.02?
> 
> Thanks for your time!
> Abel

http://mdouchin.github.io/lizmap-presentation/#/46 Unfortunately for us the relative path of threejs do not works and we use the same solution of this issue https://github.com/3liz/lizmap-web-client/issues/712 (a web link)

**Comment by RobiFag** [comment]
Hi all,
I made lots of tests but I'm not able to display an html file created with qgis2threejs plugin. I can load the .html file in the right-dock for instance but I always get the same errors i.e. all .js file created by the qgis2threejs plugin are not found. I have tried to put all the files created by the plugin more or less everywhere, in the media folder, in the media/js folder, in the media/js/project folder but the js files are not found.
I have created a js file like the one in http://mdouchin.github.io/lizmap-presentation/#/46 and I geet the errors.
I have also tried this js file:
```
lizMap.events.on({
    uicreated: function(e) {
        var indexUrl = 'media/js/concerteaux/index.html'
        var mediaRegex = /^(\/)?media\//;
        if( mediaRegex.test( indexUrl ) ){
            var mediaLink = OpenLayers.Util.urlAppend(
                lizUrls.media,
                OpenLayers.Util.getParameterString( lizUrls.params )
            );
            indexUrl = mediaLink+'&path=/'+indexUrl;
        }
        var frameSrc = mediaLink +'&path=/media/test_3d/local/index.html';
        lizMap.addDock(
            '3D',
            '3D',
            'bottomdock',
            '<iframe width="100%" height="500" src="' + indexUrl + '" frameborder="0" allowfullscreen></iframe>',
            'icon-globe'
        );
    }
});
```
but nothing changed, I got the same errors.

Any ideas? @mdouchin how can you show the 3d model as in the image http://mdouchin.github.io/lizmap-presentation/#/45?
